README.md

```diff
@@ -22,16 +23,147 @@ INSTALLED_APPS = [
+urlpatterns = [
+    ...
+    *arches_rest_framework_urls,
+]
+```
+
+For developer install instructions, see the [Developer Setup](#developer-setup-for-contributing-to-the-arches-querysets-project) section below.
 
-REST_FRAMEWORK = {
-    "DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.LimitOffsetPagination",
-    "PAGE_SIZE": API_MAX_PAGE_SIZE,
-}
+### Quickstart
+```shell
+python manage.py add_test_data
+python manage.py runserver
 ```
 
-### Usage
+#### API usage
+Log in to Arches, then visit `/api/resources/datatype_lookups` to explore the data for the ["datatype_lookups" test model](https://github.com/archesproject/arches-querysets/blob/88c284a458fbf2f4757621d0d381a9bf4ea6a96c/tests/utils.py#L24) using the [browsable API](https://www.django-rest-framework.org/topics/browsable-api/).
+
+You'll see a tree of tiles with nodegroup data grouped under an `"aliased_data"` key:
+
+<details>
+<summary>Example response</summary>
+
+Note that below "string", "number", "concept", etc. are node aliases.
+```json
+GET /api/resource/datatype_lookups
+
+HTTP 200 OK
+Allow: GET, POST, HEAD, OPTIONS
+Content-Type: application/json
+Vary: Accept
+
+{
+    "count": 2,
+    "next": null,
+    "previous": null,
+    "results": [
+        {
+            "resourceinstanceid": "486412dd-fed8-45b4-a0e0-5cd738f0eaeb",
+            "aliased_data": {
+                "datatypes_1": {
+                    "tileid": "8368dbc8-7cbc-4080-8623-46c928f011b8",
+                    "resourceinstance": "486412dd-fed8-45b4-a0e0-5cd738f0eaeb",
+                    "nodegroup": "c8d6ae9b-d9e4-4ecc-87ef-6183d84df305",
+                    "parenttile": null,
+                    "aliased_data": {
+                        "string": {
+                            "display_value": "forty-two",
+                            "interchange_value": {
+                                "en": {
+                                    "value": "forty-two",
+                                    "direction": "ltr"
+                                }
+                            }
+                        },
+                        "number": {
+                            "display_value": "42",
+                            "interchange_value": 42
+                        },
+                        "concept": {
+                            "display_value": "Arches",
+                            "interchange_value": {
+                                "concept_id": "00000000-0000-0000-0000-000000000001",
+                                "language_id": "en",
+                                "value": "Arches",
+                                "valueid": "d8c60bf4-e786-11e6-905a-b756ec83dad5",
+                                "valuetype_id": "prefLabel"
+                            }
+                        },
+                        ...
 ```
-forthcoming
+</details>
+- At the bottom of the page, switch to the JSON view to edit the payload and save back. (You can also provide interchange
```

Comment:
The formatting here doesn't get parsed properly w/o a linefeed before the bulleted list:
```suggestion

- At the bottom of the page, switch to the JSON view to edit the payload and save back. (You can also provide interchange
```